contrib/construct_dictionary.py

```diff
@@ -315,10 +319,11 @@ def main():
         return 1
 
     # mark the end of the array
-    constants.write(""",
-    {.name = ""}
+    constants.write(""",\n
+    {.index = UINT32_MAX, .name = "", .string = "", .type = PMIX_POINTER, .description = (char *[]){"NONE"}}
```

Comment:
The empty descriptions must be NULL termianted.
```suggestion
    {.index = UINT32_MAX, .name = "", .string = "", .type = PMIX_POINTER, .description = (char *[]){"NONE", NULL}}
```

Reply:
Similar change is needed for `PMIX_POINTER` on like 78